client/interfaces.go

```diff
@@ -56,6 +56,7 @@ type ContainerServer interface {
 	GetContainers() (containers []api.Container, err error)
 	GetContainer(name string) (container *api.Container, ETag string, err error)
 	CreateContainer(container api.ContainersPost) (op *Operation, err error)
+	CreateContainerFromImage(source ImageServer, image api.Image, imgcontainer api.ContainersPost) (op *RemoteOperation, err error)
```

Comment:
Hm, I would have found it nicer if we had just one single `CreateContainer()` function which we'd expose just so that the API is asimap (as simple as possible). :)